.github/workflows/cypress.yml

```diff
@@ -1,46 +1,48 @@
 name: Cypress
 
-on:
-  pull_request:
-    paths:
-      - '.github/workflows/**'
-      - 'appinfo/**'
-      - 'cypress/**'
-      - 'l10n/**'
-      - 'lib/**'
-      - 'src/**'
-      - 'skeleton/**'
-      - 'templates/**'
-      - '.php-cs-fixer.dist.php'
-      - 'babel.config.js'
-      - 'composer.json'
-      - 'composer.lock'
-      - 'cypress.json'
-      - 'package.json'
-      - 'package-lock.json'
-
-  merge_group:
-
-  push:
-    branches:
-      - main
-      - master
-      - stable*
+on: pull_request
 
 permissions:
   contents: read
 
 concurrency:
-  group: ${{ github.head_ref || github.run_id }}
+  group: cypress-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 env:
   APP_NAME: collectives
   CYPRESS_baseUrl: http://localhost:8081/index.php
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+
+    outputs:
+      src: ${{ steps.changes.outputs.src}}
+
+    steps:
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+        id: changes
+        continue-on-error: true
+        with:
+          filters: |
+            src:
+              - '.github/workflows/**'
+              - 'src/**'
+              - 'appinfo/info.xml'
+              - 'package.json'
+              - 'package-lock.json'
+              - 'tsconfig.json'
+              - '**.js'
+              - '**.ts'
+              - '**.vue'
```

Comment:
I think we should also run cypress on changes to the backen - i.e. `lib`.

Reply:
While you're at it you could make the workflow expression more precise and only run on changes to the cypress workflow.

Reply:
Well, the workflow regex is the default from all upstream templates, I'd rather keep it that way to not diverge more. But agreed, `lib` should be added. Good catch :ok_hand: